src/pages/playground/index.tsx

```diff
@@ -23,9 +25,33 @@ import { trpc } from 'lib/client/trpc';
 import type PageWithLayout from 'types/persistentLayout';
 
 const Playground: PageWithLayout = ({}) => {
+  const router = useRouter();
+
+  const isCategories = (
+    value: string | string[] | undefined
+  ): value is trpc['playground']['getAllRequests']['input']['categories'] => {
```

Comment:
I would prefer using the enum name `PlaygroundRequestCategory` directly to make this more explicit and shorter.